rca/project_styleguide/templates/patterns/organisms/programme_tabs/experience.html

```diff
@@ -0,0 +1,79 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<section class="section section--start section--end bg bg--dark">
+    <header class="section__header grid">
+        <h2 class="section__heading section__heading--primary heading heading--two">Experience</h2>
+    </header>
+    <div class="section__content">
+        
+        {% if page.experience_introduction or page.experience_cta_link %}
+            <div class="overview grid">
+                {% if page.experience_introduction %}
+                    <div class="overview__item">
+                        <div class="overview__introduction introduction">
+                            {{ page.experience_introduction }}
+                        </div>
+                    </div>
+                {% endif %}
+                
+                {% if page.experience_cta_link %}
+                    <div class="overview__item">
+                        <div class="overview__item-action">
+                            {% for block in page.experience_cta_link %}
+                                {% include_block block %}
+                            {% endfor %}
+                        </div>
+                    </div>
+                {% endif %}
+            </div>
+        {% endif %}
+
+        {% if page.experience_content %}
+            {% for block in page.experience_content %}
+                {% if block.block_type == "social_embeds" or block.block_type == "story" %}
+                    <div class="section__header grid">
+                        <h3 class="section__heading section__heading--secondary heading heading--three">{{ block.value.title }}</h3>
+                    </div>
+                    <div class="section__row grid">
+                        <div class="results layout__start-one layout__span-two layout__@large-start-two layout__@large-span-three">
+                            {% if block.block_type == "social_embeds" %}
+                                {% for individual_embed in block.value.embeds %}
+                                    <div class="results__item layout__span-two layout__@medium-span-one social-embeds__item">
```

Comment:
It looks like `layout__span-two layout__@medium-span-one` here doesn't do anything since the parent div doesn't have `grid`.

Reply:
I checked this out and it actually used to stack the embeds vertically in one column when viewing on screens smaller than tablet
<img width="288" height="454" alt="Screenshot 2025-08-01 at 11 48 32" src="https://github.com/user-attachments/assets/0b5e46c0-5485-450c-b0c4-04104086bab6" />